packages/common/lib/common/tools.js

```diff
@@ -3439,6 +3439,29 @@ async function getInstancesOrderedByStartPrio(objects, logger, logPrefix = '') {
  * @returns {Promise<void>}
  */
 async function setExecutableCapabilities(execPath, capabilities, modeEffective, modePermitted, modeInherited) {
+    // if Docker and Admin Capabilities should be set check if we are allowed to do that
+    if (isDocker() && capabilities.includes('cap_net_admin')) {
+        try {
+            const systemCaps = fs.readFileSync('/proc/$$/status', 'utf-8');
```

Comment:
I don't think that $$ for process id works here

Reply:
hehe ,,, right